EntityModel/Organization.cs

```diff
@@ -54,12 +62,29 @@ public void UpdateGender(Gender gender, bool add)
             {
                 this.Gender &= ~gender;
             }
+        }
+        public void UpdateCaseManagementGender(Gender gender, bool add)
+        {
+            if (add)
+            {
+                this.CaseManagementGender |= gender;
+            }
+            else
+            {
+                this.CaseManagementGender &= ~gender;
+            }
```

Comment:
Dedupe

Reply:
TODO: Need to find a better way to remove duplication since object member references are not supported in C#